var/spack/repos/builtin/packages/nvpl-blas/package.py

```diff
@@ -0,0 +1,69 @@
+# Copyright 2013-2023 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class NvplBlas(Package):
+    """
+    NVPL BLAS (NVIDIA Performance Libraries BLAS) is part of NVIDIA Performance Libraries
+    that provides standard Fortran 77 BLAS APIs as well as C (CBLAS).
+    """
+
+    homepage = "https://docs.nvidia.com/nvpl/_static/blas/index.html"
+    url = (
+        "https://developer.download.nvidia.com/compute/nvpl/redist"
+        "/nvpl_blas/linux-sbsa/nvpl_blas-linux-sbsa-0.1.0-archive.tar.xz"
+    )
+
+    maintainers("albestro", "rasolca")
+
+    license("UNKNOWN")
```

Comment:
@fspiga @samcmill Do you have any suggestion with respect to the license, or the PR in general?